streams/runners/storm/runtime/src/main/java/com/hortonworks/streamline/streams/runtime/storm/bolt/StreamlineHiveMapper.java

```diff
@@ -29,8 +29,8 @@
     public StreamlineHiveMapper(List<String> fields, List<String> partitionFields) {
         Objects.requireNonNull(fields, "Empty fields");
         Objects.requireNonNull(partitionFields, "Empty partitionFields");
-        this.fields = new ArrayList<>(fields);
-        this.partitionFields = new ArrayList<>(partitionFields);
+        this.fields = fields.stream().map(field -> field.substring(field.indexOf(":") + 1, field.length())).collect(Collectors.toList());
```

Comment:
No need to have endIndex  as field.length() as `field.substring(startIndex)` is sufficient. You may want to do the same change for the next line.